File: data/src/main/java/com/ykb/app/cryptotrader/data/repo/ParameterRepo.java
package com.ykb.app.cryptotrader.data.repo;

import com.ykb.app.cryptotrader.data.model.Parameter;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ParameterRepo extends JpaRepository<Parameter, String> {
}

File: library/data/data-core/src/main/kotlin/com/ykb/app/cryptotrader/data/core/repo/ParameterRepo.kt
package com.ykb.app.cryptotrader.data.core.repo

import com.ykb.app.cryptotrader.data.core.model.Parameter
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface StrategyRepo : JpaRepository<Parameter, String>